src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -37,6 +37,7 @@ object Backend {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+  type A <: BackendCall
```

Comment:
single letters starting with A is both idiomatic and suggested by the scala syle guide.